uli_init/simulate.py

```diff
@@ -349,432 +385,38 @@ def anneal(
                 dynamic=["momentum"]
             )
 
-            for kT in schedule:  # Start iterating through annealing steps
-                print(f"Running @ Temp = {kT} kT")
+            if pressure:
+                try:
+                    integrator.disable()
+                except:
```

Comment:
```suggestion
                except NameError:
```
I think this is trying to check if the integrator exists, right?

Reply:
good catch